Automattic/portkey-android
# Progress notifications: fix slide count

Fixes #345 

Builds on top of #390 

WPAndroid related PR: https://github.com/wordpress-mobile/WordPress-Android/pull/12300

Easy fix, was counting range `0..quantity` when should have been `0..quantity-1`.

To test:
1. create a story with one frame (better if its video so you get to see the notification for longer)
2. tap PUBLISH to start the saving process
3. observe the notification reads "1 slide remaining" instead of "2 slides remaining"

<img width="394" alt="Screen Shot 2020-06-26 at 12 14 08" src="https://user-images.githubusercontent.com/6597771/85872814-a80b7a00-b7a6-11ea-922f-7f040ccdee46.png">
